### ISSUE TITLE
[readability/braces] False positive in multiline macros

### ISSUE THREAD

**Issue by ptomato** [cpplint/cpplint#406]
Reproducer:
```c++
// Copyright 2025
#define MACRO         \
    if (func1())      \
        return false; \
                      \
    func2();
```
Cpplint says:
```
bug.cpp:3:  If/else bodies with multiple statements require braces  [readability/braces] [4]
```
If you delete line 5 the error doesn't occur. So I think the alignment of the `\` is confusing cpplint into thinking there is another indented statement in the body of the `if`.
